### ISSUE TITLE
Provisioning Profile Delete Error Token

### ISSUE THREAD

**Issue by julianoterres** [fastlane/fastlane#13256]
### New Issue Checklist

- [x] Updated fastlane to the latest version
- [x] I read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [x] I read [docs.fastlane.tools](https://docs.fastlane.tools)
- [x] I searched for [existing GitHub issues](https://github.com/fastlane/fastlane/issues)

### Issue Description
When I execute the method delete_provisioning_profile I have received the follow error message:

765: unexpected token at 'Bad Request'

I already fixed the error in my localhost. How I do to send this fix to repository? I made the clone of repository in mac, I created the new branch and fix the code, but when I try made a push of my branch I reciveded the folow message:

ERROR: Permission to fastlane/fastlane.git denied to julianoterres.
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.

**Comment by janpio** [comment]
Hi @julianoterres!

You best click the "fork" button on the top right to create a fork of fastlane in your namespace, there you will be able to push your fork (add the new repo as a remote to your local checkout). Then you will be able to use https://github.com/fastlane/fastlane/compare to create a new Pull Request based on that branch (or GitHub will tell you with a little box that you can create a PR for that branch automatically)

Let me know if you have any questions or trouble along the way.

**Comment by julianoterres** [comment]
@janpio Ok, I made the pull request, follow link:

https://github.com/fastlane/fastlane/pull/13258

If you could approved the pull request fast, I be grateful, because this error be causing problens to me in other macs.

thanks

**Comment by janpio** [comment]
I can't do that as I don't know enough about that code, and even when this is merged it will have to be released first before it becomes available to you.

But thanks for creating the PR, now someone can review it 👍

**Comment by janpio** [comment]
Could you please update this issue description or the PR description to include the actual (full) error that is being fixed by this? Currently we only know that you had a problem and "765: unexpected token at 'Bad Request'", which doesn't tell us enough what is going on or if your fix is appropriate. Thanks.

**Comment by julianoterres** [comment]
The error "765: unexpected token at 'Bad Request" was happening bacause the tokens not was sending in header of the post in the method "delete_provisioning_profile!". I saw that the function "repair_provisioning_profile" there were many people complaining the same error. I analyzed the code and I saw that at function "repair_provisioning_profile" removing this part of code bellow:

ensure_csrf(Spaceship::Portal::ProvisioningProfile) do

and this resolved the problem in "repair_provisioning_profile", then I apply the same fix in "delete_provisioning_profile".
